refinery/file_store/models.py

```diff
@@ -280,98 +238,33 @@ def get_file_size(self, report_symlinks=False):
             return 0
 
     def get_file_extension(self):
-        '''Return extension of the file on disk or from the source.
```

Comment:
these comments are clearer

Reply:
Perhaps. However, old comments don't match the new functionality.